restli-server/src/test/java/com/linkedin/restli/server/TestCollectionResult.java

```diff
@@ -0,0 +1,111 @@
+package com.linkedin.restli.server;
+
+import com.google.common.collect.ImmutableList;
```

Comment:
Avoid using Guava classes. We're trying to slowly remove this dependency because it's problematic.

Reply:
Sure, replace it with `Collections.unmodifiableList(Arrays.asList(FOO_1))`.